libvast/src/system/configuration.cpp

```diff
@@ -70,9 +70,30 @@ caf::error configuration::parse(int argc, char** argv) {
     caf::put(content, "vast.bare-mode", true);
     command_line.erase(it);
   }
+  // Detect when plugins or plugin-dirs are specified on the command line. This
+  // needs to happen before the regular parsing of the command line since
+  // plugins may add additional commands.
+  auto is_not_plugin_opt = [](auto& x) {
+    return !detail::starts_with(x, "--plugins=")
+           && !detail::starts_with(x, "--plugin-dirs=");
+  };
+  auto plugin_opt = std::stable_partition(
+    command_line.begin(), command_line.end(), is_not_plugin_opt);
+  auto plugin_args = std::vector<std::string>{};
+  std::move(plugin_opt, command_line.end(), std::back_inserter(plugin_args));
```

Comment:
I guess this was supposed to call a function other than `std::move()`?

Reply:
No, this is correct. There is both a `std::move` in `<utility>` and one in `<algorithm>`: https://en.cppreference.com/w/cpp/algorithm/move